sparkle/src/components/index.ts

```diff
@@ -12,7 +12,7 @@ export {
   CheckBoxWithTextAndDescription,
 } from "./Checkbox";
 export { Chip } from "./Chip";
-export { Citation } from "./Citation";
+export * from "./Citation";
```

Comment:
Any reason for "*"?

Reply:
yes, exporting `Citation` and `CitationType` (was already needed before, but missing in exports)